i18n/en.pot

```diff
@@ -146,6 +146,21 @@ msgstr ""
 msgid "View summary"
 msgstr ""
 
+msgid "Ready"
+msgstr ""
+
+msgid "Running"
+msgstr ""
+
+msgid "Failure"
+msgstr ""
+
+msgid "Done"
+msgstr ""
+
+msgid "Synchronization Status"
+msgstr ""
```

Comment:
[Minor] We show lowercase here but uppercase in the table:

![image](https://user-images.githubusercontent.com/24643/58706195-9abc6a00-83b1-11e9-9c82-2cfbd9f908c3.png)

Reply:
Also, the dropdown is very close to the title. This didn't happen before because we had the search bar. Not sure if this should/can be fixed.

Reply:
We noticed with Adrian too the spacing, will add some margin to top.